AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomGPU_BasicLevelSetup.py

```diff
@@ -0,0 +1,288 @@
+            timeout_in_seconds=4.0
+        )
+        result = isclose(
+            a=general.get_viewport_size().x, b=SCREEN_WIDTH, rel_tol=0.1) and isclose(
+            a=general.get_viewport_size().y, b=SCREEN_HEIGHT, rel_tol=0.1)
+
+        return result
+
+    with Tracer() as error_tracer:
+        # Test setup begins.
+        # Setup: Wait for Editor idle loop before executing Python hydra scripts then open "Base" level.
+        helper.init_idle()
+        helper.open_level("", "Base")
+
+        # Test steps begin.
+        # 1. Wait for the level to load and update the viewport size.
+        helper.after_level_load()
+        Report.critical_result(Tests.viewport_set, initial_viewport_setup(SCREEN_WIDTH, SCREEN_HEIGHT))
+
+        # 2. Create Default Level Entity.
+        default_level_name = "Default Level"
+        default_level_entity = EditorEntity.create_editor_entity_at(math.Vector3(0.0, 0.0, 0.0), default_level_name)
+
+        # 3. Create Grid Entity as a child entity of the Default Level Entity.
+        grid_name = "Grid"
+        grid_entity = EditorEntity.create_editor_entity(grid_name, default_level_entity.id)
+
+        # 4. Add Grid component to Grid Entity and set Secondary Grid Spacing.
+        grid_component = grid_entity.add_component(grid_name)
+        secondary_grid_spacing_property = "Controller|Configuration|Secondary Grid Spacing"
+        secondary_grid_spacing_value = 1.0
+        grid_component.set_component_property_value(secondary_grid_spacing_property, secondary_grid_spacing_value)
+        secondary_grid_spacing_set = grid_component.get_component_property_value(
+            secondary_grid_spacing_property) == secondary_grid_spacing_value
+        Report.result(Tests.secondary_grid_spacing, secondary_grid_spacing_set)
+
+        # 5. Create Global Skylight (IBL) Entity as a child entity of the Default Level Entity.
+        global_skylight_name = "Global Skylight (IBL)"
+        global_skylight_entity = EditorEntity.create_editor_entity(global_skylight_name, default_level_entity.id)
+
+        # 6. Add HDRi Skybox component to the Global Skylight (IBL) Entity.
+        hdri_skybox_name = "HDRi Skybox"
+        hdri_skybox_component = global_skylight_entity.add_component(hdri_skybox_name)
+        Report.result(Tests.hdri_skybox_component_added, global_skylight_entity.has_component(hdri_skybox_name))
+
+        # 7. Add Global Skylight (IBL) component to the Global Skylight (IBL) Entity.
+        global_skylight_component = global_skylight_entity.add_component(global_skylight_name)
+        Report.result(Tests.global_skylight_component_added, global_skylight_entity.has_component(global_skylight_name))
+
+        # 8. Set the Cubemap Texture property of the HDRi Skybox component.
+        global_skylight_image_asset_path = os.path.join(
+            "LightingPresets", "greenwich_park_02_4k_iblskyboxcm_iblspecular.exr.streamingimage")
```

Comment:
In the interest of keeping the repo size small, I suggest removing this asset from the AutomatedTesting project and using Gems\Atom\Feature\Common\Assets\LightingPresets\default_iblskyboxcm.exr instead. The greenwich exr is ~25mb, and there's really no need to add yet another skybox.

also, use the product without the _iblspecular in it, since that is what typically goes in the skybox.

Reply:
ticket cut to deal with this, out of scope after investigating